packages/flutter/lib/src/material/stepper.dart

```diff
@@ -594,6 +605,7 @@ class _StepperState extends State<Stepper> with TickerProviderStateMixin {
 
     return ListView(
       shrinkWrap: true,
+      physics: widget.physics ?? AlwaysScrollableScrollPhysics(),
```

Comment:
Are we sure we want to default this ot AlwaysScrollableScrollPhysics?  That seems off to me but I'm not sure I'm piecing all of this together correctly.

If it is right, it should be `const`.

Reply:
As other comment. Removing default.